simple-chat-front/src/components/TestComponent/TestComponent.jsx

```diff
@@ -1,16 +1,30 @@
-import React, { useState } from 'react';
-import './TestComponent-Style.css'
+import { React, useState, useEffect, Component } from "react";
+import "./TestComponent-Style.css";
+import apiRequests from "../../utils/apiRequests.js";
 
 const TestComponent = () => {
-    const [value, setValue] = useState(0);
-    return (
-        <div>
-            <h2>My testComponent</h2>
-            <div className="pressme-value">{value}</div>
-            <button className="pressme-btnp" onClick={() => setValue(value + 1)}>Press Me</button>
-            <button className="pressme-btnm" onClick={() => setValue(value - 1)}>Press Me</button>
-        </div>
-    );
-}
+  const [value, setValue] = useState(0);
+  const [data, setData] = useState(0);
 
-export default TestComponent;
\ No newline at end of file
+  useEffect(() => {
+    apiRequests().then((data) => {
+      setData(data);
```

Comment:
Change variable name, `setData` is too generic, change the name to the data that is actually set, for example: `setName` if you're setting a name